test/models/CDSAccountStatusSpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import utils.SpecBase
+import models.CDSAccountStatus
+import play.api.libs.json.{JsValue, Json, JsString, JsSuccess}
+
+class CDSAccountStatusSpec extends SpecBase {
+
+  "CDSAccountStatus" must {
```

Comment:
Could you please refactor as below cause must is already at line 25?

  "CDSAccountStatus.CDSAccountStatusReads" must {. // you could also use "CDSAccountStatus Reads" 

      "read AccountStatusOpen correctly" in new Setup {
        val res = CDSAccountStatus.CDSAccountStatusReads.reads(jsOpen)
        res mustBe JsSuccess(AccountStatusOpen)
      }

      "read AccountStatusSuspended correctly" in new Setup {
        val res = CDSAccountStatus.CDSAccountStatusReads.reads(jsSuspended)
        res mustBe JsSuccess(AccountStatusSuspended)
      }

      "read AccountStatusClosed correctly" in new Setup {
        val res = CDSAccountStatus.CDSAccountStatusReads.reads(jsClosed)
        res mustBe JsSuccess(AccountStatusClosed)
      }

      "read Unknown correctly" in new Setup {
        val res = CDSAccountStatus.CDSAccountStatusReads.reads(jsFailure)
        res mustBe JsSuccess(AccountStatusOpen)
      }
  }

Note: - you could also use like below for your test description but to read correctly implicitly means JsSuccess 

"read AccountStatusOpen correctly as jsSuccess AccountStatusOpen"